x-pack/plugins/ml/public/application/explorer/explorer.tsx

```diff
@@ -569,6 +569,7 @@ export const Explorer: FC<ExplorerUIProps> = ({
                 bounds={bounds}
                 interval={swimLaneBucketInterval ? swimLaneBucketInterval.asSeconds() : undefined}
                 chartsCount={chartsData.seriesToPlot.length}
+                queryString={queryString}
```

Comment:
Most of the props we pass here are available via [this context](https://github.com/elastic/kibana/blob/76fcb3fab99ab28ab1c1938fe767db70d04478ba/x-pack/plugins/ml/public/application/explorer/anomaly_explorer_context.tsx#L35). 
Would be better to place `queryString` in the appropriate state service and retrieve it from the context later.

Reply:
Updated here [`25da066` (#140147)](https://github.com/elastic/kibana/pull/140147/commits/25da06687b0ccb2f28ff0a117ef5eb53bbed6a4e)